ChEMBL to Solr/Database Reader (#1)/settings.xml

```diff
@@ -0,0 +1,62 @@
olregno = syn_ban.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'DCF'%%00010) syn_dcf ON md.molregno = syn_dcf.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'E_NUMBER'%%00010) syn_enum ON md.molregno = syn_enum.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'FDA'%%00010) syn_fda ON md.molregno = syn_fda.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'INN'%%00010) syn_inn ON md.molregno = syn_inn.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'JAN'%%00010) syn_jan ON md.molregno = syn_jan.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'MERCK_INDEX'%%00010) syn_merck ON md.molregno = syn_merck.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'NATIONAL_FORMULARY'%%00010) syn_nf ON md.molregno = syn_nf.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'OTHER'%%00010) syn_other ON md.molregno = syn_other.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'SYSTEMATIC'%%00010) syn_sys ON md.molregno = syn_sys.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'TRADE_NAME'%%00010) syn_trade ON md.molregno = syn_trade.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'USAN'%%00010) syn_usan ON md.molregno = syn_usan.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, synonyms FROM synonyms_agg WHERE syn_type = 'USP'%%00010) syn_usp ON md.molregno = syn_usp.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 1%%00010) alert_glaxo ON md.molregno = alert_glaxo.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 2%%00010) alert_dundee ON md.molregno = alert_dundee.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 3%%00010) alert_bms ON md.molregno = alert_bms.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 4%%00010) alert_pains ON md.molregno = alert_pains.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 5%%00010) alert_surechembl ON md.molregno = alert_surechembl.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 6%%00010) alert_mlsmr ON md.molregno = alert_mlsmr.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 7%%00010) alert_inpharmatica ON md.molregno = alert_inpharmatica.molregno%%00010LEFT JOIN (%%00010  SELECT molregno, alerts FROM alerts_agg WHERE alert_set_id = 8%%00010) alert_lint ON md.molregno = alert_lint.molregno"/>
```

Comment:
Not the friendliest presentation of SQL I've ever seen.
Easiest to load it up in KNIME and look at it.